storage/mysql/provider.go

```diff
@@ -107,3 +120,32 @@ func (s *mysqlProvider) AdminStorage() storage.AdminStorage {
 func (s *mysqlProvider) Close() error {
 	return s.db.Close()
 }
+
+func registerTLSConfig() error {
+	if *mySQLTLSCA == "" {
+		return nil
+	}
+
+	rootCertPool := x509.NewCertPool()
+	pem, err := os.ReadFile(*mySQLTLSCA)
+	if err != nil {
+		return err
+	}
+	if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
+		return errors.New("failed to append PEM")
+	}
+	serverName := ""
+	if *mySQLServerName != "" {
+		serverName = *mySQLServerName
+	} else {
+		serverName = os.Getenv("MYSQL_HOSTNAME")
+	}
```

Comment:
This isn't mentioned in the PR description/CHANGELOG or flags. 
Is this ENV var used anywhere else or is it a new one? If it's new then let's remove it for now and rely on just flags for configuration to avoid confusion.

Reply:
I thought it is used explicitly by trillian (as it is used by mySQL), but you're right, this is a new var (removed it)